keybase/platform_windows.go

```diff
@@ -58,9 +63,82 @@ func (c context) BeforeUpdatePrompt(update updater.Update, options updater.Updat
 	return nil
 }
 
+func (c config) promptProgram() (command.Program, error) {
+	destinationPath := c.destinationPath()
+	if destinationPath == "" {
+		return command.Program{}, fmt.Errorf("No destination path")
+	}
+
+	return command.Program{
+		Path: "mshta.exe",
+		Args: []string{filepath.Join(destinationPath, "prompter", "prompter.hta")},
+	}, nil
+}
+
+type updaterWinPromptInput struct {
+	Title       string `json:"title"`
+	Message     string `json:"message"`
+	Description string `json:"description"`
+	AutoUpdate  bool   `json:"autoUpdate"`
+	OutPathName string `json:"outPathName"`
+	TimeoutSecs int    `json:"timeoutSecs"`
+}
+
 func (c context) UpdatePrompt(update updater.Update, options updater.UpdateOptions, promptOptions updater.UpdatePromptOptions) (*updater.UpdatePromptResponse, error) {
-	// No update prompt for Windows, since the installer may handle it
-	return &updater.UpdatePromptResponse{Action: updater.UpdateActionContinue}, nil
+	promptProgram, err := c.config.promptProgram()
+	if err != nil {
+		return nil, err
+	}
+
+	promptJSONInput, err := c.promptInput(update, options, promptOptions)
+
+	if err != nil {
+		return nil, fmt.Errorf("Error generating input: %s", err)
+	}
+
+	// Unmarshal so we can add a couple of fields
+	var promptArgs updaterPromptInput
```

Comment:
Instead of separate prompt struct you can add OutPath to the regular one... it will be ignored if unsupported..

Put a comment next to it though saying `// Used for windows instead of stdout` or something